draft-packages/tag/docs/Tag.stories.tsx

```diff
@@ -164,3 +164,9 @@ StickerSheetReversed.parameters = {
   chromatic: { disable: false },
   controls: { disable: true },
 }
+
+export const LiveTag: StoryFn<typeof Tag> = args => (
```

Comment:
Do we definitely want another story for this? It's covered in the sticker sheet

Reply:
No, we don't. Added it just for dev convenience.